utils/wasm-gen/src/lib.rs

```diff
@@ -311,6 +316,45 @@ impl<'a> WasmGen<'a> {
         }
     }
 
+    // ~1% of cases with invalid stack size not a multiple of the page size
+    // ~1% of cases with invalid stask size that is biger than import memory
+    // ~1% of cases stack size is not generated at all
+    // all other cases should be valid
+    fn get_gear_stack_end_seed(&mut self, min_memory_size_pages: u32) -> GearStackEndExportSeed {
+        const MAX_SEED: u32 = 100;
```

Comment:
I think it's better to use Ratio and also add this 1% in Config

Reply:
done